mkosi.conf

```diff
@@ -0,0 +1,43 @@
+# A basic configuration to test azure-init
+#
+# Currently, this only works when used from an Azure VM since it relies on the
+# Azure-provided IMDS and Wireserver instances. In the future we can provide stubs
+# so it's easy to test locally or in GitHub actions. We can also add configurations
+# for additional distributions.
+#
+# To use this:
+#	- Install mkosi
+#	- Run `mkosi -d fedora` or `mkosi -d debian`
+#	- Run `sudo mkosi -d fedora boot`
+#
+# The repository is available under /work
+[Output]
+BuildDirectory=.mkosi/builddir
+CacheDirectory=.mkosi/cache
+Format=directory
+
+[Content]
+Autologin=yes
+
+Packages=
+	curl
+	gdb
+	sudo
+	systemd
+	util-linux
+
+[Host]
+# Build an image with `genisoimage -udf -o target/image.iso ovf` where ovf is a
+# directory with a valid `ovf-env.xml` file in it. Then uncomment below to add
+# the ISO as a CD-ROM in the virtual machine.
+#
+# This only works with a bootable image.
+#QemuArgs=-cdrom target/image.iso
+
+# Boot to a temporary snapshot of the image - changes won't be persisted to the
+# image.
+Ephemeral=true
+# Perform an incremental rebuild of the image
+Incremental=true
+# Make the source available at runtime
+RuntimeBuildSources=yes
```

Comment:
How about adding `ToolsTree=default` here?
On my Debian stable system with systemd 252, `mkosi boot` does not run because systemd-nspawn version is too low.

See also [NEWS](https://github.com/systemd/mkosi/blob/main/NEWS.md#v20):
`ToolsTree=default can be used to make sure a recent version of systemd is used to do the image build`.